### PR TITLE
creates bikes controller

### PR DESCRIPTION
**Pull request by rociojalifi**
Please can you review my code?

### CODE REVIEW

File: app/controllers/bikes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+class BikesController < ApplicationController
+  def index
+    @bikes = Bike.new
```
Comment (EUGENIA-AMAYA) [inline app/controllers/bikes_controller.rb:3]: here I believe should go a Bike.all, right?

File: app/controllers/bikes_controller.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  def destroy
+    set_bike
+    @bike.destroy
+    redirect_to root_path
+  end
+
+  private
+  def bike_params
+    params.require(:bike).permit(:model, :photo, :description, :price)
```
Comment (EUGENIA-AMAYA) [inline app/controllers/bikes_controller.rb:42]: location en vez de model?

File: app/controllers/bikes_controller.rb (hunk 1 above)
Comment (rociojalifi) [reply app/controllers/bikes_controller.rb]: Right! done it

File: app/controllers/bikes_controller.rb (hunk 2 above)
Comment (rociojalifi) [reply app/controllers/bikes_controller.rb]: Listo!

**Review by EUGENIA-AMAYA — APPROVED** [review]

**Pull request merged**